modules/services/kanshi.nix

```diff
@@ -0,0 +1,201 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.kanshi;
+
+  outputModule = types.submodule {
+    options = {
+
+      criteria = mkOption {
+        type = types.str;
+        description = ''
+          <literallayout>
+          The criteria can either be an output name, an output description or "*".
+          The latter can be used to match any output.
+
+          On sway(1), output names and descriptions can be obtained via
```

Comment:
Use `citerefentry` like in
https://github.com/rycee/home-manager/blob/ecb93ab9aeeb718e238114243d951a82f20ebf3f/modules/services/picom.nix#L121-L124